cmd/crank/main.go

```diff
@@ -18,38 +18,27 @@ limitations under the License.
 package main
 
 import (
-	"fmt"
-
 	"github.com/alecthomas/kong"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
 
 	"github.com/crossplane/crossplane/cmd/crank/beta"
+	v "github.com/crossplane/crossplane/cmd/crank/version"
```

Comment:
Slight preference to just import this as `version` (not `v`) if possible. As much as I often argue for short variable names, `v` is a little too short for an import IMO. 🙂

Reply:
removed